README.md

```diff
@@ -42,7 +42,13 @@ $ yarn add eslint \
   @angular-eslint/eslint-plugin-template \
   @angular-eslint/template-parser \
   @typescript-eslint/eslint-plugin \
-  @typescript-eslint/parser -D
+  @typescript-eslint/parser \
+  eslint-import-resolver-typescript \
+  eslint-plugin-deprecation \
+  eslint-plugin-import \
+  eslint-plugin-jsdoc \
+  eslint-plugin-prefer-arrow \
+  -D
```

Comment:
What does `-D` do? end of block?

Reply:
Installs them as a devDependency vs dependency